### PR TITLE
Fix DesignatedVoting commits in voter-dapp

### PR DESCRIPTION
**Pull request by mrice32**

### CODE REVIEW

File: voter-dapp/src/ActiveRequests.js [hunk 1]
```diff
@@ -255,7 +255,7 @@ function ActiveRequests({ votingAccount, votingGateway }) {
         hash: computeVoteHash({
           price,
           salt,
-          account,
+          votingAccount,
```
Comment (nicholaspai) [inline voter-dapp/src/ActiveRequests.js:258]: Just realized. This needs to be `account: votingAccount`

Comment (mrice32) [reply voter-dapp/src/ActiveRequests.js]: Oh right, didn't realize it's an object instead of an arg list.

Comment (mrice32) [reply voter-dapp/src/ActiveRequests.js]: Done. PTAL.

Comment (nicholaspai) [reply voter-dapp/src/ActiveRequests.js]: LGTM

**Pull request merged**